dev.neovide.neovide.yml

```diff
@@ -0,0 +1,233 @@
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - '*.a'
+      - '*.la'
+    modules:
+      - name: libuv
+        sources:
+          - type: archive
+            url: https://github.com/libuv/libuv/archive/v1.48.0.tar.gz
+            sha256: 8c253adb0f800926a6cbd1c6576abae0bc8eb86a4f891049b72f9e5b7dc58f33
+  - name: msgpack-c
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/msgpack/msgpack-c/releases/download/c-4.0.0/msgpack-c-4.0.0.tar.gz
+        sha256: 420fe35e7572f2a168d17e660ef981a589c9cbe77faa25eb34a520e1fcc032c8
+    cleanup:
+      - /include
+      - /lib/cmake
+      - /lib/pkgconfig
+      - '*.a'
+  - name: unibilium
+    no-autogen: true
+    make-install-args:
+      - PREFIX=/app
+    sources:
+      - type: archive
+        url: https://github.com/neovim/unibilium/archive/v2.1.1.tar.gz
+        sha256: 6f0ee21c8605340cfbb458cbd195b4d074e6d16dd0c0e12f2627ca773f3cabf1
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /share/man
+      - '*.la'
+  - name: libvterm
+    no-autogen: true
+    make-install-args:
+      - PREFIX=/app
+    sources:
+      - type: archive
+        url: https://github.com/neovim/libvterm/archive/0a15c6e983b0db7ef8276e0792414a805d01bdaf.tar.gz
+        sha256: c4683e7a2d71c04781fd0ab7719a94202800e97a9e091514c16983bb732b0fa7
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - '*.a'
+      - '*.la'
+  - name: libtermkey
+    no-autogen: true
+    make-install-args:
+      - PREFIX=/app
+    sources:
+      - type: archive
+        url: http://www.leonerd.org.uk/code/libtermkey/libtermkey-0.22.tar.gz
+        sha256: 6945bd3c4aaa83da83d80a045c5563da4edd7d0374c62c0d35aec09eb3014600
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /share/man
+      - '*.la'
+  - name: tree-sitter
+    no-autogen: true
+    make-install-args:
+      - PREFIX=/app
+    sources:
+      - type: archive
+        url: https://github.com/tree-sitter/tree-sitter/archive/c3d45a0153e2985e386dd7172dd55026bb38c9ee.tar.gz
+        sha256: 88961080f0e1413a3975a640e6e879b2233fee3353c68d129332ccf0a04ff8bc
+    cleanup:
+      - /lib/pkgconfig
+  - name: neovim
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DUSE_BUNDLED=OFF
+    sources:
+      - type: archive
+        url: https://github.com/neovim/neovim/archive/v0.10.2.tar.gz
+        sha256: 546cb2da9fffbb7e913261344bbf4cf1622721f6c5a67aa77609e976e78b8e89
+        x-checker-data:
+          is-main-source: true
+          type: anitya
+          project-id: 9037
+          url-template: https://github.com/neovim/neovim/archive/v$version.tar.gz
+  - name: host-spawn
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 host-spawn /app/bin/host-spawn
+    sources:
+      - type: file
+        url: https://github.com/1player/host-spawn/releases/download/v1.6.0/host-spawn-x86_64
```

Comment:
why not building this from source?

Reply:
copied this part from vscodium manifest

i'll try building from source

Reply:
fixed, now building from source